tests/cupy_tests/math_tests/test_arithmetic.py

```diff
@@ -228,7 +228,8 @@ def check_binary(self, xp):
         # TODO(niboshi): Fix this. If rhs is a Python complex,
         #    numpy returns complex64
         #    cupy returns complex128
-        if xp is cupy and isinstance(arg2, complex):
+        if (xp is cupy and isinstance(arg2, complex)
+                and self.name != 'float_power'):
```

Comment:
I added the missing `F` datatype but still error persisted,
Actually I had tested for `float_power` for the test cases available for `cupy.power`, I found that here dtype was **converting** to **complex64** which was causing the error. 
@emcastillo, kindly can you please review  and provide me with further guidance.

Reply:
in `float_power` everything gets promoted to `float64` but complex numbers, which gets promoted to `complex128`

Reply:
tests passed in my environment, Let me check the CIs